examples/wifi-echo/server/esp32/main/gen/callback-stub.c

```diff
@@ -1789,6 +1777,18 @@ int8_t emberAfPluginNetworkSteeringGetPowerForRadioChannelCallback(uint8_t chann
 // Ifdef out the attribute change callback, since we implement it in
 // DataModelHandler
 #if 0
```

Comment:
I realize you didn't add this, but there should be no commented-out or if-0'd code in the tree

Reply:
I will happily remove the #if 0 :)

Reply:
Done !